documentation/contributing/pullrequests/index.markdown

```diff
@@ -86,3 +86,27 @@ If the commits apply cleanly to future branches and there is no evidence that th
 Otherwise either the requestor or the maintainer should create new pull-requests targeting the later branches.
 If possible, merge these *together* with the original request.
 In this context it might be worth spending some time on making use of features available in later ROS distributions to simplify the code, e.g. by using a new coding standard or a more current version of a library.
+
+### Making a new release
+
+Release person must have a write access to both devel repos (e.g. [github.com/ros-planning/moveit](https://github.com/ros-planning/moveit)) as well as release repos (e.g. [github.com/ros-gbp/moveit-release](https://github.com/ros-gbp/moveit-release)).
+
+#### Typical protocol for a single release
+If each step ends with issues, they need to be fixed before moving on.
+
+1. Notify maintainers to freeze new merge until you finish.
+1. Plan the possible date where you finish releasing tasks. Then communicate with ROS buildfarm mainteners at https://discourse.ros.org/c/release to tell that we want soak time of one or two weeks so that maintainers/early adopters can test the binaries from [shadow repo](http://wiki.ros.org/ShadowRepository) and add necessary fix if any.
+1. Run ROS buildfarm prerelease test for **all** supported Ubuntu distribution in [REP-0003](http://www.ros.org/reps/rep-0003.html) 
```

Comment:
I added links to ROS wiki, where I just added an instruction for how to run the prerelease test (with some concrete examples using MoveIt! :P ).